src/query/interpreter.cpp

```diff
@@ -1434,27 +1519,36 @@ PreparedQuery PrepareTriggerQuery(ParsedQuery parsed_query, const bool in_explic
   auto *trigger_query = utils::Downcast<TriggerQuery>(parsed_query.query);
   MG_ASSERT(trigger_query);
 
-  auto callback = [trigger_query, interpreter_context, dba, &user_parameters,
-                   owner = StringPointerToOptional(username)]() mutable {
+  std::optional<Notification> trigger_notification;
+  auto callback = [trigger_query, interpreter_context, dba, &user_parameters, owner = StringPointerToOptional(username),
+                   &trigger_notification]() mutable {
     switch (trigger_query->action_) {
       case TriggerQuery::Action::CREATE_TRIGGER:
+        trigger_notification = Notification(SeverityLevel::INFO, NotificationCode::CREATE_TRIGGER,
+                                            fmt::format("Created trigger {}.", trigger_query->trigger_name_));
```

Comment:
Why not emplace directly?

Reply:
I've missed trigger notifications since I've removed optional...